package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "insee-brut",
+  "version": "1.0.0",
+  "description": "Mirror site for INSEE data aimed to be exhaustive, faster and easier to browse.",
```

Comment:
ah ok You customised this root package.json file, but I think it should not be here but inside the `client` folder. I'm pretty sure there should not be two different `package.json` files, one at the root level and one in the `client` folder